src/routes/projects.tsx

```diff
@@ -85,7 +87,7 @@ function ProjectsIndexRoute() {
         </Box>
       </Toolbar>
       <Card sx={{ flex: 1, minWidth: 0 }}>
-        <DataGrid
+        <DataGrid<Project>
```

Comment:
This looks weird?

Reply:
This is to provide specific type for the component. It automatically infers the column definition typings from that.